src/tests/ftest/deployment/ior_per_rank.yaml

```
@@ -0,0 +1,67 @@
+hosts:
+  test_servers: 2
+  test_clients: 2
+timeout: 1500
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+        1:
+          class: nvme
+          bdev_list: ["aaaa:aa:aa.a"]
```

We should auto here:
```suggestion
      storage: auto
```